UnitTests/UnitTests.UWP/UI/Extensions/TextBox/Test_TextBoxExtensions_Regex.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Toolkit.Uwp;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Markup;
+
+namespace UnitTests.UWP.UI
+{
+    [TestClass]
+    public class Test_TextBoxExtensions_Regex : VisualUITestBase
+    {
+        [TestCategory("TextBoxExtensionsRegex")]
+        [TestMethod]
+        public async Task Test_SetAndCheckValidationModeNormal()
+        {
+            await App.DispatcherQueue.EnqueueAsync(async () =>
+            {
+                var treeRoot = XamlReader.Load(@"<Page
+                xmlns=""http://schemas.microsoft.com/winfx/2006/xaml/presentation""
+                xmlns:x=""http://schemas.microsoft.com/winfx/2006/xaml""
+                xmlns:ui=""using:Microsoft.Toolkit.Uwp.UI"">
```

Comment:
I'm currently working on the UnitTest for TextBoxExtensionsRegex. But I have a problem with the inclusion of the extension. I get always the error, that the attachable property 'ValidationMode' was not found in type 'TextBoxExtensions'. [Line: 7 Position: 29] @jamesmcroft or @michael-hawker can you look into it? I just don't know why it doesn't work here, it's the normal syntax.